diff --git a/node_modules/jest-resolve/build/isBuiltinModule.js b/node_modules/jest-resolve/build/isBuiltinModule.js
index 8569160..03a2372 100644
--- a/node_modules/jest-resolve/build/isBuiltinModule.js
+++ b/node_modules/jest-resolve/build/isBuiltinModule.js
@@ -35,5 +35,5 @@ const BUILTIN_MODULES = new Set(
 );
 
 function isBuiltinModule(module) {
-  return BUILTIN_MODULES.has(module);
+  return BUILTIN_MODULES.has(module.startsWith('node:') ? module.slice(5) : module);
 }
